test/micro/org/openjdk/bench/java/lang/StringConstructor.java

```diff
@@ -28,47 +28,51 @@
 import java.nio.charset.StandardCharsets;
 import java.util.concurrent.TimeUnit;
 
-@Fork(5)
 @State(Scope.Thread)
 @BenchmarkMode(Mode.AverageTime)
 @OutputTimeUnit(TimeUnit.NANOSECONDS)
-@Warmup(iterations = 5, time = 5)
-@Measurement(iterations = 10, time = 5)
+@Warmup(iterations = 5, time = 3)
+@Measurement(iterations = 5, time = 3)
+@Fork(3)
 public class StringConstructor {
+
+  @Param({"0", "7", "64"})
+  public int size;
```

Comment:
I suggest to add the param `offset` for future experiment: together with `perfasm` it helps to check how different stubs are used and emulate the different branches of the optimized code

Reply:
Done